far/fnparce.cpp

```diff
@@ -129,6 +131,7 @@ namespace tokens
 		short_name_extension         = L"!-!"sv,
 		short_name_extension_safe    = L"!+!"sv,
 		current_drive                = L"!:"sv,
+		description                  = L"!*"sv,
```

Comment:
We will get a conflict here.
!* already has a meaning: ! (long file name without extension) + * (wildcard), and can be used to match all files that begin with the current file name.

Reply:
Hah, I confess I did not think of that. In hindsight, the asterisk should have felt dangerous... I suggest  `!%` instead.

Reply:
On second thought, that may conflict with environment variables.

Reply:
I have settled on `!?!` because it does not seem to have any conflicts and maybe this insert is closest in nature to the prompt in that it contains an arbitrary string; implemented in 73ad553.